db/migration/1652693645426-RemoveMinPopulationFilter.ts

```diff
@@ -0,0 +1,22 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+
+export class RemoveMinPopulationFilter1652693645426
+    implements MigrationInterface
+{
+    name = "RemoveMinPopulationFilter1652693645426"
+
+    public async up(queryRunner: QueryRunner): Promise<void> {
+        const tables = {
+            charts: "config",
+            suggested_chart_revisions: "suggestedConfig",
+        }
```

Comment:
Nice that you included `suggested_chart_revisions`! I wonder if we should also include `chart_revisions`? I haven't done it consistently in the past, but it seems somewhat safe to do in this case